packages/salesforce-adapter/src/client/client.ts

```diff
@@ -448,6 +459,14 @@ export default class SalesforceClient {
       _.defaults({}, config?.maxConcurrentApiRequests, DEFAULT_MAX_CONCURRENT_API_REQUESTS),
       SALESFORCE
     )
+    this.rateLimiters.total.once('failed', async (e, _jobInfo) => {
```

Comment:
if you are not using a parameter, don't get it
```suggestion
    this.rateLimiters.total.once('failed', async error => {
```